docs/learn/learn-polkadot-opengov-treasury.md

```diff
@@ -79,6 +79,41 @@ track on-chain. Learn how to submit a treasury proposal for referendum
 
 :::
 
+## Sub-treasuries
+
+The {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} treasury currently operates under
+a single account on-chain. The above tracks manage the treasury and represent the funds on the
+network. However, creating _sub_-treasuries that correspond to each
+[collective](./learn-system-chains#collectives) is also possible.
+
+Rather than have many referenda on the relay chain, the overarching
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Treasury can allocate funds to each
+sub-treasury (through [governance](./learn-polkadot-opengov)), from which each respective collective
+can spend funds (depending on their specific rule set).
+
+New treasuries could be added to respective
+[system chains](./learn-system-chains#existing-system-chains) through governance by adding more
+instances of this pallet.
+
+## Multi-Asset Treasury Support
+
+The use of [cross-consensus messaging](./learn-xcm.md) would also mean that these treasuries can
+support multiple asset types, and thus can spend assets other than
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} held within the treasury. These assets have a
+few requirements:
+
+1. The asset is listed on the [AssetHub system parachain](https://assethub-polkadot.subscan.io/).
+2. The asset is active and has sufficient liquidity to be utilized for payouts.
+3. The asset has a set conversion rate, as per OpenGov referenda on the Treasurer track (set via the
+   asset rate pallet). This conversion rate defines a fixed-rate representation for converting from
```

Comment:
Is this true? How can the conversion rate be fixed, given the assets are volatile?

Reply:
Sorry, I meant a **fixed** point, not a fixed rate, although I think that is the case anyway, as the rate is a `FixedI128` set in the StorageMap and can be updated.  not sure what the strategy is to maintain it unless I am misunderstanding something.

For more detail, take a look here, and how the rate is set in the pallet: https://github.com/paritytech/polkadot-sdk/blob/475e7a147676a4e7a9d255ddc7969dd35ea22882/substrate/frame/asset-rate/src/lib.rs#L18